src/localPipResolver.js

```diff
@@ -1,48 +1,43 @@
 'use strict';
 
-var logger = require('pelias-logger').get('wof-admin-lookup');
-var createPIPService = require('pelias-wof-pip-service').create;
+const logger = require('pelias-logger').get('wof-admin-lookup');
+const createPipService = require('./pip/index').create;
 
 /**
  * LocalPIPService class
  *
- * @param {object} [lookupService] optional, primarily used for testing
+ * @param {object} [pipService] optional, primarily used for testing
  * @constructor
  */
-function LocalPIPService(lookupService, datapath) {
+function LocalPipService(datapath) {
+  const self = this;
 
-  this.lookupService = lookupService || null;
+  createPipService(datapath, (err, service) => {
+     self.pipService = service;
+  });
 
-  if (!this.lookupService) {
-    var self = this;
-    createPIPService(datapath, function (err, service) {
-      self.lookupService = service;
-    });
-  }
 }
 
 /**
  * @param {object} centroid
- * @param {number} centroid.lat
- * @param {number} centroid.lon
+ * @param {array} search_layers
  * @param callback
  */
-LocalPIPService.prototype.lookup = function lookup(centroid, callback, search_layers) {
-
-  var self = this;
+LocalPipService.prototype.lookup = function lookup(centroid, search_layers, callback) {
+  const self = this;
 
   // in the case that the lookup service hasn't loaded yet, sleep and come back in 5 seconds
-  if (!self.lookupService) {
-    setTimeout(function () {
-      self.lookup(centroid, callback);
+  if (!self.pipService) {
+    setTimeout(() => {
+      self.lookup(centroid, search_layers, callback);
     }, 1000 * 5);
     return;
   }
 
-  self.lookupService.lookup(centroid.lat, centroid.lon, function (err, results) {
+  self.pipService.lookup(centroid.lat, centroid.lon, search_layers, (err, results) => {
```

Comment:
I'm so glad you rearranged the order of these params to be sane again. Having the callback not be last definitely smells.